app/api/ada/lib/utils.js

```diff
@@ -65,6 +67,48 @@ const _getTxCondition = (state: string): AdaTransactionCondition => {
   return 'CPtxWontApply';
 };
 
+export class CborIndefiniteLengthArray {
```

Comment:
I think we don’t need export for this one

Reply:
Agree. Removed